packages/x/xcb-util-errors/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 371158
+  rss: https://gitlab.freedesktop.org/api/v4/projects/xorg%2Flib%2Flibxcb-errors/repository/tags
```

Comment:
OK, I see what's happening here. You're taking the "Version check URL" from release-monitoring.org and using that as the RSS URL. That isn't correct as that's not actually an RSS feed URL (in most cases). How to get the actual one depends on how the source is hosted, Github and Gitlab for example provide them automatically:

Github-hosted projects:
- Go to the project page
- Click on "Releases" and see if the project uses releases (not every project does)
- If it does use releases then take the URL (which should end in `/releases`) and add `.atom` to it. This is your RSS URL
- If it doesn't then go to the tabs page instead and take the URL (which should end in `/tags`) and add `.atom` to it. This is your RSS URL

Gitlab-hosted projects:
- Go to the project page
- Find the "Releases" tab buried in the sidebar (not every project has that tab enabled so it might not be present)
- If it does have a releases tab and the project uses it then find the RSS/Feed icon on this page (looks kinda like a wifi/signal icon)
- Right click it and select "Copy Link". This is your RSS URL
- Sometimes Gitlab adds a tracking marker to this URL which you'll need to delete. I'm not able to find an example for this but it will look like a UUID.
- If the project doesn't use releases you can do the same for the `tags` page.